exec/cnex/lib/net.c

```diff
@@ -58,6 +58,25 @@ static SOCKET *check_socket(TExecutor *exec, Object *o)
     return o->ptr;
 }
 
+static char *socket_error_message(int err)
```

Comment:
This should return `const char` instead of `char`

Reply:
That's what was intended.  I missed that.  It's been fixed and pushed.